frontend/packages/pipelines-plugin/src/components/pipelines/list-page/PipelineAugmentRunsWrapper.tsx

```diff
@@ -14,11 +14,17 @@ interface PipelineAugmentRunsWrapperProps {
 }
 
 const PipelineAugmentRunsWrapper: React.FC<PipelineAugmentRunsWrapperProps> = (props) => {
+  const { t } = useTranslation();
   const pipelineData = _.get(props.pipeline, 'data', []);
-  if (pipelineData.length < 1) {
+
+  if (!props.pipeline.loaded) {
+    return <LoadingBox />;
+  }
+
+  if (pipelineData.length === 0) {
     return (
       <div className="cos-status-box">
-        <div className="text-center">No {PipelineModel.labelPlural} Found</div>
+        <div className="text-center">{t('pipelines-plugin~No Pipelines Found')}</div>
```

Comment:
can we use the `EmptyBox` component here ?

Reply:
Problem with EmptyBox is, it uses dynamic values (label) in the translation and generates keys in `utils.json`. More over the label in our case is `Pipelines` will not be translated.